server/memphis_handlers_user_mgmt.go

```diff
@@ -1046,3 +1049,24 @@ func (umh UserMgmtHandler) DoneNextSteps(c *gin.Context) {
 
 	c.IndentedJSON(200, gin.H{})
 }
+
+func (umh UserMgmtHandler) SkipNextStep(c *gin.Context) {
+	var body models.SkipNextStep
+	ok := utils.Validate(c, &body, false, nil)
```

Comment:
this endpoint can get the username from within the token sent with the request, take a look on the createStation endpoint for example you will find a function called getUserDetailsFRomMiddleware